NetKet/Sampler/exact_sampler.hpp

```diff
@@ -133,6 +133,11 @@ class ExactSampler : public AbstractSampler {
     }
     return acc;
   }
+
+  void SetMachineFunc(std::function<double(const Complex&)> machine_func) {
```

Comment:
Should be marked `override`. Also, see comments above regarding the empty state of `std::function`.